docs/examples/04_backends_vrep/01_simulation_with_vrep.rst

```diff
@@ -71,8 +71,8 @@ forward kinematics:
         group_a = robot_a.model.attr['index']
         group_b = robot_b.model.attr['index']
 
-        frame_a = client.forward_kinematics(robot, None, group=group_a)
-        frame_b = client.forward_kinematics(robot, None, group=group_b)
+        frame_a = client.forward_kinematics(robot_a, None, group=group_a)
+        frame_b = client.forward_kinematics(robot_b, None, group=group_b)
```

Comment:
Why this wasn't a problem before is a bit of a mystery to me...  But it's right now.